common/test_filestore_utils.py

```diff
@@ -22,26 +22,63 @@
 # the local_filestore implementation.
 
 
-def test_keyword_args():
-    """Tests that keyword args, and in particular 'parallel' are handled
-    correctly."""
-    filestore_path = 'gs://fake_dir'
-
-    with mock.patch('common.new_process.execute') as mocked_execute:
-        filestore_utils.rm(filestore_path, recursive=True, parallel=True)
-        mocked_execute.assert_called_with(
-            ['gsutil', '-m', 'rm', '-r', filestore_path], expect_zero=True)
-
-    with mock.patch('common.new_process.execute') as mocked_execute:
-        mocked_execute.return_value = new_process.ProcessResult(0, '', '')
-        filestore_utils.ls(filestore_path)
-        mocked_execute.assert_called_with(['gsutil', 'ls', filestore_path],
-                                          expect_zero=True)
-
-    filestore_path2 = filestore_path + '2'
-
-    with mock.patch('common.new_process.execute') as mocked_execute:
-        filestore_utils.cp(filestore_path, filestore_path2, parallel=True)
-        mocked_execute.assert_called_with(
-            ['gsutil', '-m', 'cp', filestore_path, filestore_path2],
-            expect_zero=True)
+class TestLocalFilestore:
+    """Tests for switching on local_filestore."""
+    FILESTORE = '/experiment_data'
+    DIR1 = '/dir1'
+    DIR2 = '/dir2'
+
+    def test_local_filestore_on(self, use_local_filestore):  # pylint: disable=unused-argument
```

Comment:
I think the name `test_using_local_filestore` is more consistent with `use_local_filestore`

Reply:
Alright.
Previously I used `switch_on` so I used `on` here.
Nvm